sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1314,6 +1314,31 @@ object SQLConf {
       "Other column values can be ignored during parsing even if they are malformed.")
     .booleanConf
     .createWithDefault(true)
+
+  val REPL_EAGER_EAVL_ENABLED = buildConf("spark.sql.repl.eagerEval.enabled")
+    .doc("Enable eager evaluation or not. If true and the REPL you are using supports eager " +
+      "evaluation, Dataset will be ran automatically. The HTML table which generated by " +
+      "<code>_repl_html_</code> called by notebooks like Jupyter will feedback the queries " +
```

Comment:
Can you rewrite these descriptions based on the description I posted in the original PR.

Reply:
Thanks, done in afada2b.